package.json

```diff
@@ -27,6 +27,6 @@
     "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-promise": "^6.1.1",
     "mocha": "^10.2.0",
-    "nodemon": "^2.0.21"
+    "nodemon": "^2.0.22"
```

Comment:
Do you know how this `nodemon` version got changed? Seems like something that should have only happened if you were maybe missing package-lock.json or told it to update somehow manually?

Reply:
It must have automatically upgraded it when I was playing around with some features, so I changed it back to the original version.

Reply:
Only PR #13 needs these script/package changes, so I'm going to remove them here. (please see [slack thread](https://codethedream.slack.com/archives/C060LAKRWTG/p1699122656551589))